src/portable/st/stm32_fsdev/dcd_stm32_fsdev.c

```diff
@@ -233,7 +235,7 @@ void dcd_init (uint8_t rhport)
   {
     pma[PMA_STRIDE*(DCD_STM32_BTABLE_BASE + i)] = 0u;
   }
-  USB->CNTR |= USB_CNTR_RESETM | USB_CNTR_SOFM | USB_CNTR_CTRM | USB_CNTR_SUSPM | USB_CNTR_WKUPM;
+  USB->CNTR |= USB_CNTR_RESETM | USB_CNTR_SOFM | USB_CNTR_ESOFM | USB_CNTR_CTRM | USB_CNTR_SUSPM | USB_CNTR_WKUPM;
```

Comment:
I think `USB_CNTR_ESOFM` should only be enabled by `dcd_remote_wakeup()` and clear when unused. It will prevent MCU is interrupted when in suspended mode, and haven't got any need to wake up yet. This will help to application with low power design. Sorry for being picky, If you don't have time, I will merge and change it myself, not a problem at all. Your work is already fantastic